src/math/Box3.js

```diff
@@ -244,29 +244,13 @@ Box3.prototype = {
 
 	containsPoint: function ( point ) {
 
-		if ( point.x < this.min.x || point.x > this.max.x ||
-				 point.y < this.min.y || point.y > this.max.y ||
-				 point.z < this.min.z || point.z > this.max.z ) {
-
-			return false;
-
-		}
-
-		return true;
+		return !(point.x < this.min.x || point.x > this.max.x || point.y < this.min.y || point.y > this.max.y || point.z < this.min.z || point.z > this.max.z);
```

Comment:
Thanks for your reactivity !
I will implement it in few minutes, but just before do this, a little question.

Why would you keep `=== false` at the end of the line ? 
For readability ? Because with:

```javascript
return !( point.x < this.min.x || point.x > this.max.x || point.y < this.min.y || point.y > this.max.y || point.z < this.min.z || point.z > this.max.z );
```

You know immediately that the statement must be the inverse to be true. And finally, the negate statement (!) is really alot faster than using the comparison with false... Almost 50-60% faster.

You could run this bench:

```javascript
var pointA     = {
    x: 5,
    y: 7,
    z: 1
};
var pointB     = {
    x: 98,
    y: 15,
    z: 7
};
var iterations = 10000000;

function comparisonFunction() {
    return ( pointA.x < pointB.x || pointA.x > pointB.x || pointA.y < pointB.y || pointA.y > pointB.y || pointA.z < pointB.z || pointA.z > pointB.z ) === false;
}

function negationFunction() {
    return !( pointA.x < pointB.x || pointA.x > pointB.x || pointA.y < pointB.y || pointA.y > pointB.y || pointA.z < pointB.z || pointA.z > pointB.z );
}

for (var a = 0, numberOfBench = 10; a < numberOfBench; ++a) {
    var t0 = performance.now();
    for (var i = 0; i < iterations; ++i) {
        comparisonFunction()
    }
    var t1 = performance.now();
    console.log("Call to comparisonFunction took " + (t1 - t0) + " milliseconds.");

    var t2 = performance.now();
    for (var j = 0; j < iterations; ++j) {
        negationFunction()
    }
    var t3 = performance.now();
    console.log("Call to negationFunction took " + (t3 - t2) + " milliseconds.");

    console.log(" ");
}
```

But the goal of this question is more to be sure that in future you REALLY WANT this type of statement with comparison operator, or if you prefer performance while keeping the readability with negate statement ?

Best regards,
Tristan